azure/services/scalesets/scalesets_test.go

```diff
@@ -623,6 +623,25 @@ func TestReconcileVMSS(t *testing.T) {
 				s.Location().AnyTimes().Return("test-location")
 			},
 		},
+		{
```

Comment:
Would it be possible to add positive test cases here for valid boot diagnostics? I see that you added a lot more test cases for machines, and I think the same can be applied here.

Reply:
Added 👍